### ISSUE TITLE
Benin 10-digit number validation update

### ISSUE THREAD

**Issue by pmt-nauhand** [jackocnr/intl-tel-input#1895]
### Plugin version
v24.8.1 

### Steps to reproduce
1.  Integrate the intl-tel-input plugin into a web page.
2.  Select Benin as country (+229).
3.  Try entering a 10-digit number, for example: 22991234567.

### Expected behaviour
Benin's numbers should accept 10 digits, according to the new regulations.

### Actual behaviour
The plugin limits Benin numbers to 8 digits, making it impossible to enter a 10-digit number.

### Initialisation options
```
const input = document.querySelector("#phone");

const iti = window.intlTelInput(input, {
  initialCountry: "bj",
  utilsScript: "https://cdn.jsdelivr.net/npm/intl-tel-input@latest/build/js/utils.js",
  onlyCountries: ["bj", "fr", "us"],
  separateDialCode: true,
});
```

**Comment by jackocnr** [comment]
I've just released v24.8.2 which updates libphonenumber to the latest version. Give that a try and let me know if that solves the issue.

**Comment by pmt-nauhand** [comment]
Thank you for your update: the new 10-digit format has been taken into account for Benin.
